interfaces/apparmor/apparmor.go

```diff
@@ -0,0 +1,91 @@
+// -*- Mode: Go; indent-tabs-mode: t -*-
+
+/*
+ * Copyright (C) 2016 Canonical Ltd
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License version 3 as
+ * published by the Free Software Foundation.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ *
+ */
+
+// apparmor contains primitives for working with apparmor.
+//
+// References:
+//  - http://wiki.apparmor.net/index.php/Kernel_interfaces
+//  - http://apparmor.wiki.kernel.org/
+//  - http://manpages.ubuntu.com/manpages/xenial/en/man7/apparmor.7.html
+package apparmor
+
+import (
+	"fmt"
+	"os"
+	"os/exec"
+	"strings"
+)
+
+// AddOrReplaceProfile loads an apparmor profile from the given file.
```

Comment:
Oh sure, I *did* mean that the simplified rename won't hurt since there's just one place where this is used and whoever reads that can check it out from there. Fixed